jumanji/training/configs/config.yaml

```
@@ -1,12 +1,12 @@
 defaults:
     - _self_
-    - env: snake  # [bin_pack, cleaner, connector, cvrp, game_2048, graph_coloring, job_shop, knapsack, maze, minesweeper, mmst, multi_cvrp, robot_warehouse, rubiks_cube, snake, sudoku, tetris, tsp]
+    - env: sokoban  # [bin_pack, cleaner, connector, cvrp, game_2048, graph_coloring, job_shop, knapsack, , minesweeper, mmst, multi_cvrp, robot_warehouse, rubiks_cube, snake, sudoku, tetris, tsp, sokoban]
```

Can we revert the configs? Also, it seems maze got left out
```suggestion
    - env: snake  # [bin_pack, cleaner, connector, cvrp, game_2048, graph_coloring, job_shop, knapsack, maze, minesweeper, mmst, multi_cvrp, robot_warehouse, rubiks_cube, snake, sudoku, tetris, tsp, sokoban]
```